src/Commands/SetShooter.cpp

```diff
@@ -0,0 +1,14 @@
+#include "SetShooter.h"
+#include "../Subsystems/Shooter.h"
+std::shared_ptr<Shooter> shooter = Shooter::getInstance();
+SetShooter::SetShooter(double speed) :
+		mSpeed(speed) {
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(Robot::chassis.get());
+	Requires(Shooter::getInstance().get());
```

Comment:
That would be used if I wanted to set the speed to what it currently is, this will be setting the new speed